src/revocation.rs

```diff
@@ -8,9 +8,99 @@ use crate::error::{Error, Result};
 use crate::secure_mount;
 
 use std::path::Path;
+use std::process::Command;
 
 use serde_json::Value;
 
+/// Runs a python script (used for revocation actions)
+pub(crate) fn run_python(dir: &Path, script: &str) -> Result<()> {
+    let status =
+        // TODO: Should we assume that these script names are all correctly 
```

Comment:
My $0.02: We should assume the end in `.py` as in the future it's _possible_ there could be actions that are in other languages. AFAIK there is no plans for that but I think assuming `.py` for python scripts that are run directly via interpreter is reasonable.

Reply:
Sounds good to me. I will assume if there are any non-Python scripts, they will be sorted appropriately in `run_revocation_actions()` in the future.